client/run.go

```diff
@@ -254,6 +258,27 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 		}
 	}
 
+	for _, v := range cmdVols {
+		volRef, err := parseVolumes(v)
+		if err != nil {
+			return "", err
+		}
+		// Bind mount's driver is 'vfs' so far
+		var vol = pod.UserVolume{}
+		vol.Name = volRef.Volume
+		vol.Source = volRef.Volume
+		vol.Driver = "vfs"
```

Comment:
when vol.Source is "", vol.Driver should be "", the hyperd will use its default volume driver